src/Microsoft.SqlTools.ServiceLayer/Agent/Contracts/AgentNotebookRequest.cs

```diff
@@ -0,0 +1,51 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+using Microsoft.SqlTools.ServiceLayer.TaskServices;
+using Microsoft.SqlTools.ServiceLayer.Utility;
+using Microsoft.SqlTools.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer.Agent.Contracts
+{
+    /// <summary>
+    /// SQL Agent Notebooks activity parameters
+    /// </summary>
+    public class AgentNotebooksParams : GeneralRequestDetails
+    {
+        public string OwnerUri {get; set;}
+    
+    }
+    /// <summary>
+    /// SQL Agent Notebook activity result
+    /// </summary>
+    public class AgentNotebooksResult : ResultStatus
+    {
+        public NotebookJobInfo[] Notebooks {get; set;}
+    }
+    /// <summary>
+    /// SQL Agent Notebook request type
+    /// </summary>
+    public class AgentNotebooksRequest
+    {
+        /// <summary>
+        /// Request definition
+        /// </summary>
+        public static readonly
+        RequestType<AgentNotebooksParams, AgentNotebooksResult> Type = 
+        RequestType<AgentNotebooksParams, AgentNotebooksResult>.Create("agent/notebooks");
+    }
+    /// <summary>
+    /// SQL Agent Notebook Job Info Class
+    /// </summary>
+    public class NotebookJobInfo
```

Comment:
we shouldn't define the Info classes in the contracts..they should be defined in with the info classes.  For this simple class probably sticking next to AgentJobInfo makes sense

Reply:
hmm, looks like AgentJobInfo is in contracts. I guess I'm thinking of the Data classes.  In any case, I'd stick next to AgentJobInfo.